src/tarski/reachability/gringo.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import sys
+from clingo.application import Application, clingo_main
+
+"""
+A wrapper to emulate the default clingo app behavior
+"""
+class WrapperClingo(Application):
+    def __init__(self, name):
+        self.app_name = name
```

Comment:
Is this attribute being used anywhere?

Reply:
That attribute should be `program_name`. That's a typo on my part. It is a member of the `Application` class.